accelerator/config.c

```diff
@@ -38,10 +38,22 @@ static int get_conf_key(char const* const key) {
 }
 
 status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
-  if (value == NULL && (key != CACHE && key != PROXY_API && key != QUIET && key != NO_GTTA)) {
-    ta_log_error("%s\n", "SC_NULL");
-    return SC_NULL;
+  if (value == NULL) {
```

Comment:
I like such refactoring, which improves both readability and performance (compiler optimization with lookup table). Did you figure out more occurrences for similar code?